src/Controls/src/Core/Font.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+
+namespace Microsoft.Maui.Controls
+{
+	public struct Font
```

Comment:
Bit yucky that we have the same name in different assemblies. Will be annoying. Maybe we can change the name in the base/core type?

Reply:
Moved back to having a single Font struct; we can figure out what we want to do with NamedSize at the Core level later.